democracy/views/comment.py

```diff
@@ -60,13 +60,13 @@ def _check_may_comment(self, request):
         parent = self.get_comment_parent()
         try:
             may_comment = parent.may_comment(request)
+            if not may_comment:  # pragma: no cover
```

Comment:
The name of the function is confusing as `may` imply True/False answer. Maybe renaming it to `check_commentable` would be better, then removing the `if not` check.